docs/api/admin.yaml

```diff
@@ -40,6 +40,8 @@ paths:
     $ref: './paths/admin/db-_user-.yaml'
   '/{db}/_user/{name}':
     $ref: './paths/admin/db-_user-name.yaml'
+  '/{db}/_user/{name}/all_channels':
```

Comment:
The endpoint doesn't have a leading underscore for `all_channels`, but the path $ref on the next line does.